charts/sonarqube-dce/README.md

```diff
@@ -473,12 +473,6 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `httproute.labels`           | (Optional) List of extra labels to add to the HttpRoute                                                       | `None`  |
 | `httproute.rules`            | (Optional) Extra Rules block of the HttpRoute. A default one is created with SonarWebContext and service port | `None`  |
 
-### Elasticsearch
-
-| Parameter                       | Description                                     | Default |
-| ------------------------------- | ----------------------------------------------- | ------- |
-| `elasticsearch.bootstrapChecks` | Enables/disables Elasticsearch bootstrap checks | `true`  |
```

Comment:
Why are you removing this instead of marking it as deprecated?

Reply:
As explained in person: this parameter is not in the `values.yaml`, so it makes sense to remove already.